httpmw/logging/logging_test.go

```diff
@@ -0,0 +1,52 @@
+package logging
+
+import (
+	"bytes"
+	"log/slog"
+	"net/http"
+	"net/http/httptest"
+	"time"
+
+	"github.com/gorilla/mux"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Httpmw", func() {
+	var (
+		router *mux.Router
+	)
+
+	BeforeEach(func() {
+		router = mux.NewRouter()
+	})
+
+	Describe("LoggingMiddleware", func() {
+		It("should log and return OK status", func() {
+			buf := new(bytes.Buffer)
+			slogLogger := slog.New(slog.NewJSONHandler(buf, nil))
+			router.Use(NewLogging(slogLogger))
+
+			router.HandleFunc("/", func(w http.ResponseWriter, e *http.Request) {
```

Comment:
e *http.Request

maybe use _ to show it is not used.

Reply:
done